model/src/main/kotlin/utils/FindingsMatcher.kt

```diff
@@ -128,27 +128,27 @@ class FindingsMatcher(
         val paths = (licenseFindingsByPath.keys + copyrightFindingsByPath.keys).toSet()
         val rootLicenses = getRootLicenses(licenseFindings)
 
-        val locationsForLicenses = licenseFindings
+        val locationsByLicense = licenseFindings
```

Comment:
Commit message: For consistency with what?

Reply:
With the `match` function in that same file.

Reply:
With the two other analog variables with that same function a few lines above.

Reply:
Done.

Reply:
Done.